.github/workflows/tests.yml

```diff
@@ -109,7 +109,29 @@ jobs:
             components: clippy
       - uses: Swatinem/rust-cache@v2
 
-      - run: cargo clippy
+      - run: cargo clippy -- -D warnings
```

Comment:
guessing this is analogous to `--Werror`?

Are there any other flags we should turn on?

Reply:
Yeah, its just "deny all warnings". I think the defaults are relatively sensible TBH.